funnel/forms/project.py

```diff
@@ -39,22 +39,26 @@ class ProjectForm(forms.Form):
         validators=[forms.validators.DataRequired()],
         filters=[forms.filters.strip()],
     )
-    location = forms.StringField(
-        __("Location"),
-        validators=[forms.validators.DataRequired(), forms.validators.Length(max=50)],
-        filters=[forms.filters.strip()],
-        description=__("Eg. Bangalore, Mumbai, Pune"),
-    )
     tagline = forms.StringField(
         __("Tagline"),
         validators=[forms.validators.DataRequired(), forms.validators.Length(max=250)],
         filters=[forms.filters.strip()],
         description=__("One line description of the project"),
     )
-    description = forms.MarkdownField(
-        __("Project description"),
-        validators=[forms.validators.DataRequired()],
-        description=__("Landing page contents"),
+    location = forms.StringField(
+        __("Location"),
+        description=__(
+            '“Online”, “Bangalore”, “Chennai”, “Pune”, etc (without quotes)'
+        ),
+        validators=[
+            forms.validators.DataRequired(
+                __("If this project doesn’t have a fixed location, use “Online”")
+            ),
+            forms.validators.Length(
+                min=3, max=80, message=__("%(max)d characters maximum")
+            ),
```

Comment:
A validator for quotes in the location, since people will submit `“Online”` or `"Online"` instead of `Online`.